### PR TITLE
Add support for custom point-rendering

### PR DESCRIPTION
**Pull request by evancharlton**
Allow points to have custom rendering through the `pointRenderer`
option. This property can be set on:

  1. DataProvider
  2. Collection
  3. Series

As with other properties, the most-specific item's property value will
be used.

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-198.surge.sh

**Comment by valerii-cognite** [comment]
I believe the function should return either a standard point passed into or override it

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-198.surge.sh

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-198.surge.sh

File: src/components/Points/index.js [hunk 1]
```diff
@@ -53,9 +59,24 @@ const Points = ({
   pointWidthAccessor,
   strokeWidth,
 }) => {
-  const getX = x => boundedSeries(xScale(x));
-  const getY = y => boundedSeries(yScale(y));
-  const points = data.map(d => {
+  if (!drawPoints) {
+    // We should never wind up in this situation, but just in case...
+    return null;
```
Comment (valerii-cognite) [inline src/components/Points/index.js]: If `drawPoints` is `false`, it seems instead of `default rendering`, null will be rendered

Comment (evancharlton) [reply src/components/Points/index.js]: Which it should, right? `false` should result in no points being rendered.

Comment (evancharlton) [reply src/components/Points/index.js]: Oh nevermind -- I see the issue.

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-198.surge.sh

**Pull request merged**